src/library_formatString.js

```diff
@@ -4,6 +4,12 @@
  * SPDX-License-Identifier: MIT
  */
 
+
+#if BOOTSTRAPPING_STRUCT_INFO
+assert(!LibraryManager.library);
+LibraryManager.library = {}
```

Comment:
`;` at end of line

Reply:
Also please copy over the 3-line comment from the deleted code, as without that this looks a bit mysterious.

Reply:
Done.